compiler/src/dmd/tocvdebug.d

```diff
@@ -159,7 +161,11 @@ uint cv4_Denum(EnumDeclaration e)
             EnumMember sf = (*e.members)[i].isEnumMember();
             if (!sf)
                 continue;
-            const value = sf.value().toInteger();
+
+            auto valueExp = sf.value();
+            if (auto castExp = valueExp.isCastExp())
+                valueExp = new IntegerExp(castExp.loc, castExp.e1.toInteger(), castExp.type);
```

Comment:
This is a forcible cast which assumes its argument and the new type are workable. It would be better to find out where the CastExp is applied, and do a call to optimize() there. Further, creating an AST node for an integer just so that integer can be converted to an integer is kinda going around the horn.

Reply:
> It would be better to find out where the CastExp is applied, and do a call to optimize() there.

Yeah, that's a more sensible approach (and is now pushed up). Thanks for telling me about `optimize`.